brave-spark-framework-filter/README.md

```diff
@@ -0,0 +1,39 @@
+# brave-spark-framework-filter
+
+The module contains a Spark framework filter implementing to deal with server
+side integration: Getting existing span/trace state from request,
+create and submit span with `sr`', `ss` annotations.
+
+
+Here is a use Example:
+
+Add a dependency:
+
+```
+<dependency>
+    <groupId>io.zipkin.brave</groupId>
+    <artifactId>brave-spark-framework-filter</artifactId>
+    <version>4.0.6</version>
+</dependency>
+        
+```
+
+Use it in Spark:
+
+```
```

Comment:
note..  in code blocks you can add a formatting hint right after the triple back-tick ex. ```java